datasource_api_client/models/datasource_dto_auth_type.py

```diff
@@ -10,6 +10,7 @@ class DatasourceDtoAuthType(str, Enum):
     BASIC = "Basic"
     CLIENTIDSECRET = "ClientIdSecret"
     GCPBASIC = "GCPBasic"
+    NOAUTH = "NoAuth"
```

Comment:
not sure if done through the `make update-datasource` but should also be auto-generated